stable/jenkins/values.yaml

```diff
@@ -27,6 +27,12 @@ Master:
   SlaveListenerPort: 50000
   LoadBalancerSourceRanges:
   - 0.0.0.0/0
+# List of plugins to be install during Jenkins master start
+  InstallPlugins:
+      - kubernetes:0.11 
+      - workflow-aggregator:2.5
+      - credentials-binding:1.11
+      - git:3.2.0
```

Comment:
I use GitLab instead of GitHub, so I don't need `pipeline-github-lib`. I don't think it should be installed by default